<launch>

  <node name="orb_slam3_mono" pkg="orb_slam3_ros_wrapper" type="orb_slam3_ros_wrapper_mono" output="screen">

      <remap from="/camera/image_raw" to="/camera_undistorted/image"/>

      <!-- Parameters for original ORB-SLAM3 -->
      <param name="voc_file"      type="string"   value="/root/ORB_SLAM3/Vocabulary/ORBvoc.txt" />
      <param name="settings_file" type="string"   value="$(find orb_slam3_ros_wrapper)/config/kitti_camera_color_left_right.yaml" />

      <!-- Parameters for ROS -->
      <param name="world_frame_id"    type="string"   value="world" />
      <param name="cam_frame_id"      type="string"   value="camera" />
      <param name="enable_pangolin"      type="bool"   value="false" />
  </node>

  <node pkg="hector_trajectory_server" type="hector_trajectory_server" name="trajectory_server_orb_slam3" output="screen" ns="orb_slam3_ros" >
      <param name="/target_frame_name" value="/world" />
      <param name="/source_frame_name" value="/camera" />
      <param name="/trajectory_update_rate" value="20.0" />
      <param name="/trajectory_publish_rate" value="20.0" />
  </node>
</launch>
